dockstore-integration-testing/src/test/java/io/dockstore/webservice/SearchResourceIT.java

```diff
@@ -141,11 +143,11 @@ public void testElasticSearchHealthCheck() {
         // Update the search index
         extendedGa4GhApi.toolsIndexGet();
         waitForIndexRefresh(0, extendedGa4GhApi,  0);
-        // Should still fail even with index
         try {
             metadataApi.checkElasticSearch();
+            fail("Should fail even with index because there's no hits");
         } catch (ApiException ex) {
-            assertTrue("Should fail", true);
+            Assert.assertTrue(ex.getMessage().contains("Internal Server Error"));
```

Comment:
nitpick: think the superfluous Assert adds a Codacy alert

Reply:
ahh